modules/src/ics02_client/client_def.rs

```diff
@@ -106,12 +113,39 @@ impl Header for AnyHeader {
 
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub enum AnyClientState {
-    Tendermint(crate::ics07_tendermint::client_state::ClientState),
+    Tendermint(TendermintClientState),
 
     #[cfg(test)]
     Mock(MockClientState),
 }
 
+impl AnyClientState {
+    pub fn from_any(any: prost_types::Any) -> Result<Self, Error> {
+        match any.type_url.as_str() {
+            "ibc.tendermint.ClientState" => {
```

Comment:
Minor:

Maybe we should consider keeping these strings (lines 125 `"ibc.tendermint.ClientState"` and 135 `"ibc.mock.ClientState"`) hardcoded in some enum as opposed to hardcoded here? I can imagine they could even go in the ibc-proto repo, not here, but not sure what would be best.

To keep things simpler and get this merged fast, we can also just keep the hardcoded values as they are and just add a todo.

Reply:
Yeah good point! Those should probably be defined as constants somewhere.